asb-native/build.gradle

```diff
@@ -0,0 +1,29 @@
+plugins {
+    id 'java'
+    id "com.github.johnrengelman.shadow" version "5.2.0"
+}
+
+group 'org.ballerinalang'
+version '0.1.4'
```

Comment:
Shall we point to the gradle.properties like following

```
group = project.group
version = project.version
```

Reply:
Resolved in ef99a8b. Thank u.